src/NetAppFiles/NetAppFiles/Backups/NewNetAppFilesBackup.cs

```diff
@@ -118,7 +118,7 @@ public class NewAzureRmNetAppFilesBackup : AzureNetAppFilesCmdletBase
         public string VolumeResourceId { get; set; }
 
         [Parameter(
-            Mandatory = true,            
+            Mandatory = false,            
```

Comment:
Is this an expected change? PR comments only mentioned adding missing parameter SnapshotName.

Reply:
Intended, we spoted this during this fix  the `label` parameter is not requred.